src/sentry/flags/docs/api.md

```diff
@@ -45,10 +44,11 @@ Retrieve a collection of flag logs.
     "data": [
       {
         "action": "created",
-        "flag": "my-flag-name",
         "created_at": "2024-01-01T05:12:33",
         "created_by": "2552",
         "created_by_type": "id",
+        "flag": "my-flag-name",
+        "id": 1,
```

Comment:
is the `id` the org id?

Reply:
(should it be added to the **Attributes** section above?)

Reply:
oh, looking at the code again it's the `flag log id` - it might be useful to clarify this in the blueprint

Reply:
Will do!